app/assets/javascripts/webauthn_credentials.js

```diff
@@ -0,0 +1,187 @@
+$(() => {
+  const FORM_SELECTOR = ".js-new-webauthn-credential-form"
+  const NICKNAME_INPUT_SELECTOR = `${FORM_SELECTOR} #webauthn_credential_nickname`
+  const SUBMIT_SELECTOR = `${FORM_SELECTOR} input[type=submit]`
+  const SESSION_FORM_SELECTOR = ".js-webauthn-create-session-form"
+  const SESSION_SUBMIT_SELECTOR = `${SESSION_FORM_SELECTOR} input[type=submit]`
+  const csrfToken = document.querySelector("[name='csrf-token']").content
+
+  // base64urlToBuffer and bufferToBase64url are from @github/webauthn-json
+  //
+  // Copyright (c) 2019 GitHub, Inc.
+  //
+  // Permission is hereby granted, free of charge, to any person
+  // obtaining a copy of this software and associated documentation
+  // files (the "Software"), to deal in the Software without
+  // restriction, including without limitation the rights to use,
+  // copy, modify, merge, publish, distribute, sublicense, and/or sell
+  // copies of the Software, and to permit persons to whom the
+  // Software is furnished to do so, subject to the following
+  // conditions:
+  //
+  // The above copyright notice and this permission notice shall be
+  // included in all copies or substantial portions of the Software.
+  //
+  // THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+  // EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+  // OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+  // NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+  // HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+  // WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+  // FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+  // OTHER DEALINGS IN THE SOFTWARE.
+  const base64urlToBuffer = (baseurl64String) => {
```

Comment:
I would like to `import` those functions but not sure how to that in the current setup

Reply:
Third-party code should remain in vendor/assets. You should be able to import it by adding `require ..` in application.js